source/background.ts

```diff
@@ -127,3 +127,20 @@ browser.permissions.onAdded.addListener(async permissions => {
 		await webextAlert('Refined GitHub is not meant to run on every website. If you’re looking to enable it on GitHub Enterprise, follow the instructions in the Options page.');
 	}
 });
+
+const FIREFOX_OPTIONS_MENU_ID = 'FIREFOX_OPTIONS';
+
+function onContextMenuClick({menuItemId}: Menus.OnClickData): void {
+	if (menuItemId === FIREFOX_OPTIONS_MENU_ID) {
+		void browser.runtime.openOptionsPage();
+	}
+}
+
+if (isFirefox()) {
+	browser.contextMenus.onClicked.addListener(onContextMenuClick);
+	browser.contextMenus.create({
+		id: FIREFOX_OPTIONS_MENU_ID,
+		title: 'Options',
+		contexts: ['browser_action'],
+	});
+}
```

Comment:
This also adds a shortcut like in Chrome because both Safari and Firefox make it too difficult to reach the options

<img width="398" alt="Screenshot 3" src="https://github.com/refined-github/refined-github/assets/1402241/d69c1d43-6c27-46a0-a4ec-25b0a308f045">

Normally:

- Firefox: 4 clicks
- Safari: 3 or 4 clicks if you have multiple profiles

Now:

- 2 clicks anywhere (if the `action` button is already on screen)

Reply:
Extract from https://togithub.com/pixiebrix/pixiebrix-extension/pull/847